manualTest/README.md

```diff
@@ -0,0 +1,49 @@
+# Manual Testing
```

Comment:
I'd maybe rename the manualTest dir demoSetup or sampleSetup

Reply:
Done